### PR TITLE
Fix(165,169): Participants percentage & Current vote bar

### LINKED ISSUE #165: [BUG] Participants votes section does not show the correct percentage
The votes section on the governance proposal page should display each user's percentage of the total votes for that type.

Currently, the percentage is showing 0%.

Image

### LINKED ISSUE #169: [BUG] Abstain missing from the proposal vote progress bars
## Overview

Abstain is missing representation in the Proposal page current vote section

### PR DESCRIPTION
**Pull request by RyRy79261**
## Description

This PR fixes the percentage calculation in the participant section as well as adds Abstain, to the voting indicator bars

## Related issues

- closes #165 
- closes #169

### CODE REVIEW

**Review by chapati23 — CHANGES_REQUESTED** [review]
same issue here, can't navigate to proposal details page i.e. https://governance-k1ebo4d11-mentolabs.vercel.app/proposals/76852543598626010309184355681141482743859825585547539168614458156040950326608

regardless of wallet connected/disconnected I always get console errors

image

File: src/components/votes-list/votes-list.component.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ const getParticipantPercentage = (
   totalVotes: bigint,
 ) => {
   // If there are no votes, we need to avoid division by zero
-  if (totalVotes > BigInt(0)) {
-    return `${Number((participant.weight / totalVotes) * BigInt(100)).toFixed(2)}%`;
+  if (totalVotes > 0n) {
```
Comment (denviljclarke) [inline src/components/votes-list/votes-list.component.tsx:61]: Sometimes this doesn't add up to 100%

http://localhost:3000/proposals/82815049005223249369267239582287945782658271640355079075142781978263042166954

& 

http://localhost:3000/proposals/104083448305537669896457683783159222014868105836664782870518099656159827114092

**Comment by denviljclarke** [comment]
Nice one. Left a comment about vote percentages. Looks good otherwise 👍

File: src/components/votes-list/votes-list.component.tsx (hunk 1 above)
Comment (RyRy79261) [reply src/components/votes-list/votes-list.component.tsx:61]: What values are being used?

Comment (RyRy79261) [reply src/components/votes-list/votes-list.component.tsx:61]: This line is checking if the total is above zero though

Comment (RyRy79261) [reply src/components/votes-list/votes-list.component.tsx:61]: This function gets a percentage of a given value.

Comment (chapati23) [reply src/components/votes-list/votes-list.component.tsx:61]: he's right, this one has 52% FOR and 47% AGAINST = 99% in total https://governance-5yck8hqr3-mentolabs.vercel.app/proposals/82815049005223249369267239582287945782658271640355079075142781978263042166954

Comment (RyRy79261) [reply src/components/votes-list/votes-list.component.tsx:61]: Was a rounding error, scaling for resulting percentage to have mantissa values fixed it.

**Review by philbow61 — APPROVED** [review]

**Review by bayological — APPROVED** [review]

**Pull request merged**